<?xml version="1.0"?>

<robot name="mkz" xmlns:xacro="http://www.ros.org/wiki/xacro" >

  <!-- Necessary includes -->
  <xacro:include filename="$(find dbw_mkz_description)/urdf/vehicle_structure.urdf.xacro" />
  <xacro:include filename="$(find dbw_mkz_description)/urdf/vehicle_gazebo.urdf.xacro" />

  <!-- Bolt-on sensors go here -->
  <xacro:include filename="$(find dbw_mkz_description)/urdf/vehicle_sensors.urdf.xacro" />

  <!-- Add Surround View Cameras -->
  <xacro:momenta_surround_cam name="camera_a00" parent_link="base_footprint" x="3.85" y="0" z="0.55" roll="0" pitch="0.5236" yaw="0" />
  <xacro:momenta_surround_cam name="camera_a01" parent_link="base_footprint" x="2.28" y="-0.93" z="0.88" roll="0" pitch="0.7854" yaw="-1.5708" />
  <xacro:momenta_surround_cam name="camera_a02" parent_link="base_footprint" x="-1.15" y="0" z="0.75" roll="0" pitch="0.9" yaw="3.1416" />
  <xacro:momenta_surround_cam name="camera_a03" parent_link="base_footprint" x="2.28" y="0.93" z="0.88" roll="0" pitch="0.7854" yaw="1.5708" />


</robot>

